src/imgproc.rs

```diff
@@ -89,6 +116,22 @@ pub enum HistogramComparisionMethod {
     KullbackLeiblerDivergence = 5,
 }
 
+/// ThresholdTypes used in
+/// [threshold](../struct.Mat.html#method.threshold).
+#[repr(C)]
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
+#[allow(non_camel_case_types, missing_docs)]
+pub enum ThresholdType {
+    THRESH_BINARY = 0, // \f[\texttt{dst} (x,y) =  \fork{\texttt{maxval}}{if \(\texttt{src}(x,y) > \texttt{thresh}\)}{0}{otherwise}\f]
```

Comment:
Please, provide more "rustish" names (see others enums as example).

If these comments are valuable, please make them doc comments that they appear in documentation.

Reply:
They were from the original OpenCV definitions. Actually there's no point in duplicating them here.

Reply:
Our overall intent was providing greater user expirience. If this is public enum as is intended to be used from rust code then it should use rust enum guidelines. This is why all public enums works this way. Please, use existing project code style.